### ISSUE TITLE
AcquireTokenSilent(scopes, null) is not solved by compiler (ambiguous)

### ISSUE THREAD

**Issue by jmprieur** [AzureAD/microsoft-authentication-library-for-dotnet#1051]
**Which Version of MSAL are you using ?**
Note that to get help, you need to run the latest version. Preview version are also ok. 
For ADAL, please log issues to https://github.com/AzureAD/azure-activedirectory-library-for-dotnet

**Platform**
net45

**What authentication flow has the issue?**
* Desktop / Mobile
    * [ x] Interactive
    * [ ] Integrated Windows Auth
    * [ ] Username Password
    * [ ] Device code flow (browserless)
* Web App
    * [ ] Authorization code
    * [ ] OBO
* Web API
    * [ ] OBO

Other? - please describe;

**Is this a new or existing app?**
See https://github.com/Azure-Samples/active-directory-b2c-dotnet-desktop
in branch jenny/v3

**Repro**

```CSharp
authResult =  await app.AcquireTokenSilent(App.ApiScopes, null)
 ```

**Expected behavior**
this builds? or not?

**Actual behavior**

```text
Error	CS0121	The call is ambiguous between the following methods or properties: 'IClientApplicationBase.AcquireTokenSilent(IEnumerable<string>, IAccount)' and 'IClientApplicationBase.AcquireTokenSilent(IEnumerable<string>, string)'	active-directory-b2c-wpf	c:\gh\u3\b2c-wpf\active-directory-b2c-wpf\MainWindow.xaml.cs	95	N/A
```

**Possible Solution**

**Additional context/ Logs / Screenshots**
Add any other context about the problem here, such as logs and screebshots. Logging is described at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging

**Comment by jmprieur** [comment]
@bgavrilMS I believe we discussed this?

**Comment by jmprieur** [comment]
Fixed in 3.0.4-preview
